src/Report/CDR/CDR.vue

```diff
@@ -0,0 +1,267 @@
+<template>
```

Comment:
Could you please keep formatting consistent with other similar files?

Reply:
ok, so for erlang we are using tabs (size 2) and for javascript I should use 2 spaces? Right?

Reply:
At the moment tabs for Erlang files and 2 spaces for Vue.